lib/dependencies/CommonJsRequireDependencyParserPlugin.js

```diff
@@ -73,53 +74,54 @@ class CommonJsRequireDependencyParserPlugin {
 				parser.state.current.addDependency(dep);
 				return true;
 			});
-		parser.hooks.call
-			.for("require")
-			.tap("CommonJsRequireDependencyParserPlugin", expr => {
-				if (expr.arguments.length !== 1) return;
-				let localModule;
-				const param = parser.evaluateExpression(expr.arguments[0]);
-				if (param.isConditional()) {
-					let isExpression = false;
-					const prevLength = parser.state.current.dependencies.length;
-					const dep = new RequireHeaderDependency(expr.callee.range);
-					dep.loc = expr.loc;
-					parser.state.current.addDependency(dep);
-					for (const p of param.options) {
-						const result = processItem(expr, p);
-						if (result === undefined) {
-							isExpression = true;
-						}
-					}
-					if (isExpression) {
-						parser.state.current.dependencies.length = prevLength;
-					} else {
-						return true;
+		new MultiHook([
```

Comment:
Use two `tap` calls with a arrow function as local var.

Using MultiHook would be technically valid here, but it's better to stay consitent to the other places in the code base, which call `tap` multiple times.

The original use case for the MultiHook is also different and it's a kind of abuse of it to use it this way.